src/mem/corealloc.h

```diff
@@ -506,6 +591,24 @@ namespace snmalloc
       return small_alloc_slow<zero_mem>(sizeclass, fast_free_list, rsize);
     }
 
+    /**
+     * Accessor for the local state.  This hides whether the local state is
+     * stored inline or provided externally from the rest of the code.
+     */
+    SNMALLOC_FAST_PATH
+    LocalState& get_local_state()
```

Comment:
This name is not descriptive enough, as `local_state` makes no reference to this being the `backend`'s state.  Can you change this to `get_backend_local_state`, and ensure the comments mention `backend`.